drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -4047,7 +4049,9 @@ Color RasterizerStorageGLES3::multimesh_instance_get_color(RID p_multimesh, int
 	MultiMesh *multimesh = multimesh_owner.getornull(p_multimesh);
 	ERR_FAIL_COND_V(!multimesh, Color());
 	ERR_FAIL_INDEX_V(p_index, multimesh->size, Color());
-	ERR_FAIL_COND_V(multimesh->color_format == VS::MULTIMESH_COLOR_NONE, Color());
```

Comment:
This is the line that gets spammed when saving the multimesh.

Maybe we only want to remove this line only, but I removed the other message too since it seems to be a valid color, but I have not done any 3D stuff -- I'm not too sure.